task_DEV_3/task_DEV_3/Program.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+
+namespace task_DEV_3
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Enter the size of the array :");
+            int arrayLength = Convert.ToInt32(Console.ReadLine());
+
+            if (arrayLength <= 5)
+            {
+                int[] numberArray = new int[arrayLength];
+                for (int i = 0; i < arrayLength; i++)
+                {
+                    Console.WriteLine($"Enter an array element № {i}");
+                    numberArray[i] = Convert.ToInt32(Console.ReadLine());
+                    if (numberArray[i] < 0)
+                    {
+                        Console.WriteLine("This is a negative number");
+                        break;
```

Comment:
After this check the next part of code will still process `numberArray`.

Reply:
done